src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
Don't begin class or method implementation with an empty line. There should be no empty line between the class declaration and the first member or method.